ui/src/components/harvest-games/HarvestGameInstructions.js

```diff
@@ -0,0 +1,357 @@
+import {
+  Box,
+  Container,
+  Select,
+  Heading,
+  Input,
+  Button,
+  VStack,
+  Image,
+  HStack,
+  Grid,
+} from '@chakra-ui/react';
+import { Text } from '@chakra-ui/layout';
+import { useState, Fragment } from 'react';
+
+import './styles.css';
+
+export default function HarvestGameInstructions() {
+  const gameType = [
+    {
+      game_num: 'Game #1',
+      name: 'SMUGGLING THROUGH SECURITY',
+      round: 3,
+      player: 5,
+      points: 45,
+      materials: '-',
+      imageUrl: [
+        'https://hongkong.sub.hmcc.net/wp-content/uploads/hg1_1.gif',
```

Comment:
i guess this means that all the images are hardcoded in beforehand? so the team has to provide all this before sat night

Reply:
When I replace the image on the wordpress, the link still remains the same. So I think we can change even after we deploy it. So it isn't really hard coded.